clock.js

```diff
@@ -0,0 +1,35 @@
+function currentTime() {
+  // Time variables
+  let date = new Date();
+  let hours = date.getHours();
+  let minutes = date.getMinutes();
+  let session = "AM";
+
+  let timeDoc = document.querySelector("#time");
+
+  // Changing 0 to 12 and AM/PM
+  if (hours === 0) {
+    hours = 12;
+  }
+
+  if (hours > 12) {
```

Comment:
as an alternative you could use inbuild js method: [toLocaleTimeString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleTimeString)